src/index.jsx

```diff
@@ -1,7 +1,11 @@
+import React from 'react';
 import ReactDom from 'react-dom';
 import { App } from './App';
+import { Sum } from './components/Sum/Sum';
 
 ReactDom.render(
-  <App />,
+  <App>
+    <Sum />
+  </App>,
```

Comment:
redundant change - revert it
the App component doesn't render any [children](https://react.dev/learn/passing-props-to-a-component#passing-jsx-as-children)